src/Avalonia.Themes.Fluent/Accents/BaseDark.xaml

```diff
@@ -4,13 +4,13 @@
   <Style.Resources>
     <!-- Accent Colours -->
```

Comment:
I believe this commented code isn't needed anymore.

Reply:
yeah, I'm gonna delete it